playbooks/common/openshift-cluster/upgrades/v3_5/master_config_upgrade.yml

```diff
@@ -0,0 +1,16 @@
+---
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.admission_plugin_config }}"
+  when: "{{ 'admission_plugin_config' in openshift.master }}"
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginOrderOverride'
+    yaml_value:
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'kubernetesMasterConfig.admissionConfig'
+    yaml_value:
```

Comment:
Is this just the 3.4 hook file? You can remove this file and both references to it as a hook above, then re-introduce when/if 3.5 specific steps are required. Because we require you to be on 3.4 already to run this we know these changes have already been applied.